### PR TITLE
journal-of-the-marine-biological-association-of-the-united-kingdom.csl

### PR DESCRIPTION
**Pull request by rafazildod**
Created from APA 6th edition.

This journal uses a particular set of instructions, they are relatively similar to the APA one with a handful of changes.

### CODE REVIEW

File: journal-of-the-marine-biological-association-of-the-united-kingdom.csl [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+          <else>
+            <text macro="title"/>
+          </else>
+        </choose>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author" font-style="normal">
+      <name form="short" and="symbol" delimiter-precedes-et-al="never" et-al-min="3" initialize-with=". "/>
```
Comment (rmzelle) [inline journal-of-the-marine-biological-association-of-the-united-kingdom.csl]: @rafazildod, every `et-al-min` attribute should be accompanied by a `et-al-use-first` attribute. (the first controls after how many authors "et-al"-abbreviation kicks in, and the latter determines how many names are shown before "et al." in when "et-al"-abbreviation is activated.

File: journal-of-the-marine-biological-association-of-the-united-kingdom.csl [hunk 2]
```diff
@@ -0,0 +1,599 @@
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
```
Comment (rmzelle) [inline journal-of-the-marine-biological-association-of-the-united-kingdom.csl]: @rafazildod, this style should probably be fixed to British English (`default-locale="en-GB"`), see https://github.com/citation-style-language/styles/wiki/Style-Requirements#3---style-locale.

File: journal-of-the-marine-biological-association-of-the-united-kingdom.csl [hunk 3]
```diff
@@ -0,0 +1,599 @@
...
+    <title-short>JMBA</title-short>
+    <id>http://www.zotero.org/styles/journal-of-the-marine-biological-association-of-the-united-kingdom</id>
+    <link href="http://www.zotero.org/styles/journal-of-the-marine-biological-association-of-the-united-kingdom" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
+    <link href="http://assets.cambridge.org/MBI/MBI_ifc.pdf" rel="documentation"/>
+    <author>
+      <name>Rafael Dantas</name>
+      <email>recifersd@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
```
Comment (rmzelle) [inline journal-of-the-marine-biological-association-of-the-united-kingdom.csl]: @rafazildod, can you add a line with `<category field="biology"/>` here?

Comment (rafazildod) [inline journal-of-the-marine-biological-association-of-the-united-kingdom.csl]: All suggested changes were made. :)

**Comment by rafazildod** [comment]
@rmzelle All suggested changes were made! :)

**Comment by rmzelle** [comment]
Great, thanks!

**Pull request merged**